pkg/oob/oob.go

```diff
@@ -35,11 +35,12 @@ type BMCResetter interface {
 
 // SetPower interface function for power actions
 func SetPower(ctx context.Context, action string, m []PowerSetter) (result string, err error) {
+LOOP:
```

Comment:
I'm confused what the break-LOOP does here?

Reply:
breaks out of the outer for loop instead of just breaking out of the select
https://stackoverflow.com/questions/25469682/break-out-of-select-loop

Reply:
Yeah, but is that a bug fix or break-early feature? Is it really necessary for the "check state and power on if off" case? Or does it belong in a separate commit?

Reply:
neither, it's just fixing bad code that can lead to unexpected behavior.

Reply:
I guess I'm confused why the code in server/grpcsvr/oob/machine/machine.go isn't enough to handle what the PR says its doing.

Reply:
ok gotcha that clears things up for me thanks